subsys/bluetooth/audio/tbs.c

```diff
@@ -1000,6 +1002,8 @@ static uint8_t tbs_hold_call(struct tbs_service_inst *inst,
 		return BT_TBS_RESULT_CODE_STATE_MISMATCH;
 	}
 
+	notify_calls(inst);
```

Comment:
The same applies here. Adding `notify_calls` may sent double notifications.

Reply:
Ok, I have done the changes. Thanks for your comment.